lib/matplotlib/tests/test_lines.py

```diff
@@ -85,22 +85,6 @@ def test_set_line_coll_dash():
     ax.contour(np.random.randn(20, 30), linestyles=[(0, (3, 3))])
 
 
-def test_invalid_line_data():
```

Comment:
we should should not delete this test as the first half is testing a differnt code path (that we can not pass invalid inputs to the `__init__`).